changes/21345-profile-aggregate-lodaing

```diff
@@ -0,0 +1 @@
+- fix loading state for the profile status aggregate UI
```

Comment:
This API call can be pretty heavy on the database. What do you think about reducing the number of retries as well?

Reply:
This is already set not to retry the request if there is an issue

https://github.com/fleetdm/fleet/blob/95207a4925ab879db4b846d032bed247299e1b40/frontend/pages/ManageControlsPage/OSSettings/OSSettings.tsx#L40-L52